addons/website/static/src/js/utils.js

```diff
@@ -15,19 +34,28 @@ var qweb = core.qweb;
 function autocompleteWithPages(self, $input) {
     $input.autocomplete({
         source: function (request, response) {
-            return self._rpc({
-                model: 'website',
-                method: 'search_pages',
-                args: [null, request.term],
-                kwargs: {
-                    limit: 15,
-                },
-            }).then(function (exists) {
-                var rs = _.map(exists, function (r) {
-                    return r.loc;
+            if (request.term[0] === '#') {
+                loadAnchors(request.term).then(function (anchors) {
+                    response(anchors);
```

Comment:
Should not you filter the anchors according to the user term here ? Or is the spec explicitly says it should be like this ?

Reply:
Well, specs do not seem to be strict on this part. I guess the user should see at least every existing anchors when he types '#'.

Reply:
Fine for now